httpobs/tests/unittests/test_content.py

```diff
@@ -183,7 +183,22 @@ def test_same_origin(self):
         self.assertEquals(result['result'], 'sri-not-implemented-but-all-scripts-loaded-from-secure-origin')
         self.assertTrue(result['pass'])
 
-        # On the same second-level domain
+        # On the same second-level domain, but without a protocol
+        self.reqs['resources']['__path__'] = """
+        <html>
+            <head>
+              <script src="//www.mozilla.org/static/js/foo.js"></script>
+            </head>
+            <body></body>
+        </html>
+        """
+
```

Comment:
Would you be able to push these into HTML files in the `unittests/files` directory, please?

You should be able to load them as:

`self.reqs = empty_requests('test_parse_http_equiv_headers_referrer1.html')`

Reply:
Opened #330 to track this request, will suspend work here until it's merged.

Reply:
#330 is ready to merge, woot!